README.md

```diff
@@ -11,8 +11,14 @@
 [docs-badge]: https://img.shields.io/badge/docs-latest-blue.svg
 [docs-url]: https://docs.rs/color-eyre
 
-A custom context for the [`eyre`] crate for colorful error reports, suggestions,
-and [`tracing-error`] support.
+A custom context for the [`eyre`] crate for colorful error reports with suggestions, custom
+sections, [`tracing-error`] support, and backtraces on stable.
+
+## TLDR
+
+`color_eyre` lets you easily build error reports that look like this:
```

Comment:
Suggestion: don't use words like "easily" or "simply" in documentation. If it's actually easy or simple, that's self-evident to the reader. If it's not, it just makes them more frustrated.